xlive/Blam/Engine/interface/hud.cpp

```diff
@@ -162,14 +162,15 @@ void hud_render_player_indicators(datum player_index)
 				is_enemy = game_engine_team_is_enemy((e_game_team)current_player->properties[0].team_index, (e_game_team)player->properties[0].team_index);
 			*/
 
-			bool is_enemy = game_engine_team_is_enemy((e_game_team)current_player->properties[0].team_index, (e_game_team)player->properties[0].team_index);
-			datum current_index = player_iterator.get_current_datum_index();
+			bool is_enemy = player->properties[0].team_index != player->properties[0].team_index;
```

Comment:
Revert back to the game engine function, that is a class member function for game engines

Reply:
changed the code to how it originally was